string entityId
string componentId
string componentJSON
bool isComponentMutable
